lib/diamonds.ts

```diff
@@ -0,0 +1,98 @@
+import { Contract } from 'ethers';
+
+export interface Facet {
+  target: string;
+  selectors: string[];
+}
+
+enum FacetCutAction {
+  Add,
+  Replace,
+  Remove,
+}
+
+export interface FacetCut extends Facet {
+  action: FacetCutAction;
+}
+
+export function getContractSignatures(contract: Contract) {
```

Comment:
Should these functions have an explicit return type?

Reply:
good thinking, these should definitely have return types defined.